packages/example/remotion.config.js

```diff
@@ -0,0 +1,4 @@
+import {Config} from 'remotion';
+
+Config.Rendering.setConcurrency((require('os').cpus() || {length: 1}).length);
```

Comment:
`require('os').cpus()` should never return a falsy value, so why is it needed?

Reply:
cause on my system this was giving error  `os.cpus: Cannot read property 'length' of undefined` so I added this code.

Reply:
Can you reproduce this in the console?

$ node
> require('os').cpus().length

Node.JS docs don't mention that this can happen: https://nodejs.org/api/os.html#os_os_cpus
So this is a Node.JS bug or what? 🤔